README.md

```diff
@@ -37,7 +37,8 @@ $ kubectl get pods --namespace=deis
 Once this is done, you can SSH into the minion running the controller and run the following:
 
 ```
-$ curl -sSL http://deis.io/deis-cli/install.sh | sh
+$ curl -sSL https://s3-us-west-1.amazonaws.com/bacongobbler/bacongobbler/workflow/19/19.1/client/deis
```

Comment:
Yeah this is the intention when we get this merged. This was just temporary to get other devs unblocked from testing v2. Once this is merged and the relevant envvars are set in Travis, builds will start showing up in get-deis and I'll change this over.